rest/endpoints/user_recordings/profile_recordings.yml

```
@@ -71,9 +77,48 @@ paths:
           example: 1526648593
           schema:
             type: integer
+        - in: query
+          name: filter[starts_after]
+          required: false
+          description: Include only recordings having broadcast start time after specified time (in Unix timestamp).
+          example: 1526648593
+          schema:
+            type: integer
+        - in: query
+          name: filter[ends_before]
+          required: false
+          description: Include only recordings having broadcast end time before specified time (in Unix timestamp).
+          example: 1526648593
+          schema:
+            type: integer
+        - in: query
+          name: filter[ends_after]
+          required: false
+          description: Include only recordings having broadcast end time after specified time (in Unix timestamp).
+          example: 1526648593
+          schema:
+            type: integer
+        - in: query
+          name: filter[series_id]
+          required: false
+          description: Consider only recordings having the same series_id.
+          example: "abdf8785-cb23-445b-946f-69e8770500a1"
+          schema:
+            type: string
+            nullable: true
+        - in: query
+          name: filter[channel_id]
+          required: false
+          description: Consider only recordings having the same channel_id.
+          example: "abdf8785-cb23-445b-946f-69e8770500a1"
+          schema:
+            type: string
         - in: query
           name: sort
-          description: Specifies sorting of the results. Supported fields to sort by `created_at`. If sort field is prepended with a minus sign, the order is descending. Default is ascending by `created_at` field. Example `sort=-created_at`
+          description: >-
+            Please see http://jsonapi.org/format/#fetching-sorting
+            Specifies sorting of the results. If sort field is prepended with a minus sign, the order is descending. Default is ascending by `created_at` field.
+            The following are attributes to sort on: [created_at|broadcast_slots.start_time_unix]
```

The relationship name is `broadcast_slot`
```suggestion
            The following are attributes to sort on: [created_at|broadcast_slot.start_time_unix]
```